src/log-analytics-solution/azext_log_analytics_solution/azext_metadata.json

```diff
@@ -0,0 +1,5 @@
+{
+    "azext.isExperimental": true,
+    "azext.minCliCoreVersion": "2.0.67",
+    "azext.maxCliCoreVersion": "3.0.0"
```

Comment:
- Please don't set maxCliCoreVersion here 
- "azext.minCliCoreVersion" should be at least "2.3.0"; otherwise 'experimental' tag is not available. Please test if 'experimental' tag works first.

Reply:
> Please don't set maxCliCoreVersion here
> "azext.minCliCoreVersion" should be at least "2.3.0"

Done

Reply:
>  otherwise 'experimental' tag is not available. Please test if 'experimental' tag works first.

Yeah, tag has been tested and can be used normally